pkgs/by-name/in/install-grub-ng/src/main.rs

```diff
@@ -0,0 +1,38 @@
+mod builder;
+mod config;
+mod grub;
+
+use std::path::Path;
+
+use eyre::{Result, bail};
+use roxmltree::Document;
+
+use crate::{builder::Builder, config::Config};
+
+fn main() -> Result<()> {
+    let mut args = std::env::args();
+    let Some(config_file) = args.nth(1) else {
+        bail!("Config file not given: expected it to be the first argument")
+    };
+    let Some(default_config) = args.next() else {
+        bail!("Default config not given: expected it to be the second argument")
```

Comment:
Short and (hopefully) clearer messages:
```suggestion
        bail!("Missing argv[1]: config file")
    };
    let Some(default_config) = args.next() else {
        bail!("Missing argv[2]: default config file")
```

Reply:
IMO this is too brief — we don't live in an era constrained by storage sizes anymore so might as well give the user longer but more helpful error messages